epub33/overview/index.html

```diff
@@ -142,15 +188,16 @@ <h3>Reading Order</h3>
 
 					<p>Many types of publication have an obvious reading order, or logical progression through their
 						content. A novel is an example of a highly sequential document — it typically has a beginning,
-						middle and end — but not all publications are so ordered: a cookbook or collection of
+						middle, and end — but not all publications are so ordered: a cookbook or a collection of
 						photographic images might be considered to be more like a database. All documents do, however,
 						have at least one logical ordering of all their top-level content items, whether by date, topic,
-						location, or some other criteria (e.g., a cookbook is typically ordered by type of recipe).</p>
+						location, or some other criteria (e.g., a cookbook is typically ordered by the type of a recipe).</p>
```

Comment:
"arranged by" probably makes more sense here than "ordered by". Could also be shortened to:

> (e.g., a cookbook is typically arranged by recipe type).

Reply:
Yep.